src/js/info.js

```diff
@@ -42,15 +35,15 @@ class VideoInfo {
 
 	update () {
 		if (this.brandEl) {
-			this.brandEl.textContent = formatBrand(this.video.videoData.tags);
+			this.brandEl.textContent = this.video.videoData.brand.name;
 		}
 
 		if (this.titleEl) {
 			this.titleEl.textContent = this.video.videoData.name;
```

Comment:
...or do you mean you already checked and these changes you've suggested will bring us closer to the CAPI model?